extensions/shared/components/block-styles-selector/index.js

```diff
@@ -13,21 +13,31 @@ import { BlockPreview } from '@wordpress/block-editor';
 import { useSelect } from '@wordpress/data';
 import { ENTER, SPACE } from '@wordpress/keycodes';
 
+const addPreviewAttribute = block => {
+	return {
+		...block,
+		attributes: {
+			...block.attributes,
+			isBlockPreview: true,
```

Comment:
Since this is kinda "internal" attribute, should it have underscore prefix to indicate it;  `__isblockPreview` ?

Reply:
Yes, good idea, I'll update that